script/ConfiguredScript.sol

```diff
@@ -30,13 +30,12 @@ abstract contract ConfiguredScript is Script {
     function _loadConfig(string memory network, bool requireMorpho) internal returns (bytes memory) {
         configPath = string.concat("script/", _scriptDir(), "/config/", network, ".json");
 
-        string memory latestRunPath =
-            string.concat("broadcast/DeployMorpho.sol/", vm.toString(block.chainid), "/run-latest.json");
+        string memory morphoAddressPath = string.concat("script/", _scriptDir(), "/config/", network, "-morpho.json");
 
-        if (vm.exists(latestRunPath)) {
-            string memory latestRun = vm.readFile(latestRunPath);
+        if (vm.exists(morphoAddressPath)) {
+            string memory file = vm.readFile(morphoAddressPath);
 
-            morpho = IMorpho(latestRun.readAddress("$.transactions[0].contractAddress"));
```

Comment:
What is the reason behind this change? `DeployMorpho` is only expected to be run and committed once on each chain, so `run-latest` always contains the correct address

Whereas now we rely on the dev input in `ethereum-morpho.json`, which was something you wanted to avoid at all costs @MerlinEgalite

We could also force the script to write `ethereum-morpho.json`, but it is redundant with `chainid/run-latest.json` to me
Btw, I would instead name it `/1/morpho.json`

Reply:
Yes but is that I though that first deployment was not correct (but it was) so I unfortunately overriden the `run-latest` with the 2nd deployment thus losing the data.

Reply:
I don't really mind about the naming here, I let you push something if you think it's worth it

Reply:
Given this context, what I'd recommend is:
- delete all irrelevant runs
- replace `run-latest.json` with the relevant past run (`run-latest` is always just a copy of the run with corresponding id)

I can take care of it, give me green light

Reply:
green light but the prio tbh

Reply:
Created this issue #39